### DISCUSSION TITLE
License of the source code?

### DISCUSSION THREAD

**Discussion by pago** [macabeus/vscode-fluent#15] [General]
Hello!
I stumbled upon this extension while trying to find out how to get a decent editing experience for Fluent files.

I was wondering whether you could clarify the license of the source code so that the contained syntax files could be used for other editors/IDEs as well? For example, I generally prefer to use WebStorm over VSCode and, given the syntax file, could create a TextMate bundle that would help with supporting Fluent files in WebStorm. Nowhere near as good as this extension, but decent enough for my current needs.

Do you think it would be possible to open source the code? Or, if not all of it, then perhaps the syntax file?

**Comment by macabeus** [comment]
I really forgot to add a license file.
I just committed it now. Since it's MIT, you can do your own extension using my code.
It's not mandatory, but I only ask that you open your extension when published =]

Thank you!

**Reply by pago** [reply]
Thanks a lot. 😄 
I uploaded the TextMate bundle here: https://github.com/pago/fluent.tmBundle
Seems to work fine to get me syntax highlighting in WebStorm.
